SethBurns/coloRandom
# add listeners and data model

## What is the feature?
Clicking the button will now display a random set hex colors and names

## How you implemented the solution?
Event listeners!

## How to test the feature (A test scenario or any new setup)?
- Pull branch to local and type `open index.html`
- Ensure that only elements of the button area are being used by the event listener
- Ensure valid hexes are being populated
